compiler/src/main/java/dagger/internal/codegen/BindingGraph.java

```diff
@@ -55,6 +57,8 @@
 import com.google.common.collect.Sets;
 import com.google.common.collect.TreeTraverser;
 import com.google.common.util.concurrent.ListenableFuture;
+import com.sun.tools.javac.code.Symbol;
```

Comment:
We can't use any `javac` code, it wouldn't work in any other compiler (e.g. [`ecj`](https://en.wikipedia.org/wiki/Free_Java_implementations)